### ISSUE TITLE
Error message on upgrade pb_get_seo_meta_elements & pb_get_microdata_elements

### ISSUE THREAD

**Issue by beckej13820** [pressbooks/pressbooks#1630]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

After upgrading to Pressbooks 5.7.0 and McLuhan 2.8.0 I am seeing new error messages on 1 book. Other books on the network seem uneffected.

Notice
: pb_get_seo_meta_elements is
deprecated
since version 5.7.0 with no alternative available. in
/var/www/html/wp-includes/functions.php
on line
4332

Notice
: pb_get_microdata_elements is
deprecated
since version 5.7.0 with no alternative available. in
/var/www/html/wp-includes/functions.php
on line
4332

### Steps to Reproduce

I cannot reproduce. This issue shows up on only one book after upgrade.

**Expected behavior:** 

**Actual behavior:** 

#### Book Info

Book ID: 27
Book URL: https://oer.ed-beck.com/virtuallearningdesigndelivery/
Book Privacy: Public

#### Browser

Platform: OS X
Browser Name: Chrome
Browser Version: 72.0.3626.121
User Agent String: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36

#### WordPress Configuration

Network URL: http://oer.ed-beck.com/
Network Type: Subdirectory
Version: 5.1.1
Language: en_US
WP_ENV: production
WP_DEBUG: Enabled
Memory Limit: 1000M

#### Pressbooks Configuration

Version: 5.7.0
Book Theme: McLuhan
Book Theme Version: 2.8.0
Root Theme: Aldine
Root Theme Version: 1.6.0

#### Pressbooks Dependencies

Epubcheck: Installed
Kindlegen: Installed
xmllint: Installed
PrinceXML: Installed
Saxon-HE: Installed

#### Must-Use Plugins

hm-autoloader.php: n/a

#### Network Active Plugins

GitHub Updater: 8.7.2
Openstax Import for Pressbooks: 1.2.4
Pressbooks: 5.7.0
Pressbooks LTI Provider: 1.2.0
Really Simple SSL: 3.1.5

#### Book Active Plugins

GitHub Updater: 8.7.2
Openstax Import for Pressbooks: 1.2.4
Pressbooks: 5.7.0
Pressbooks LTI Provider: 1.2.0
Really Simple SSL: 3.1.5

#### Inactive Plugins

Akismet Anti-Spam: 4.1.1
CC Export for Pressbooks: 1.0.0
Equation Editor: 1.4
H5P: 1.13.1
Hello Dolly: 1.6
Hypothesis: 0.5.0
mPDF for Pressbooks: 3.2.4
Pressbooks Stats: 1.4.1
Textbooks for Pressbooks: 4.2.3
WP QuickLaTeX: 3.8.6
Wp Show Site ID: 1.0.2
WP Super Cache: 1.6.4

#### Server Configuration

PHP Version: 7.2.15-0ubuntu0.18.04.1
MySQL Version: 5.7.25
Webserver Info: Apache/2.4.29 (Ubuntu)

#### PHP Configuration

Safe Mode: Disabled
Memory Limit: 1000M
Upload Max Size: 2000M
Post Max Size: 2500M
Upload Max Filesize: 2000M
Time Limit: 120
Max Input Vars: 1000
URL-aware fopen: On (1)
Display Errors: On (1)

#### PHP Extensions

OPcache: Zend
XDebug: Disabled
cURL: Supported
cURL Version: 7.58.0
imagick: Installed
xsl: Installed

**Comment by connerbw** [comment]
The error is: 

+ `pb_get_seo_meta_elements` is deprecated

If you have any custom code that uses that feature, then the code should be changed.

If the code is coming from one of our themes, can you look deeper and get me the line number of the actual function call? Look in your server logs for more info.

**Comment by beckej13820** [comment]
This book was running McLuhan. There were a series of weird errors coming from this book only. Chapter 1 of the book was not respecting it's "numberless" chapter type on the web version or exports. I got the error message that made me open the ticket, and the table of contents in PDF exports was not indenting consistently with other chapters.

All of the above issues are gone now with a restart, and can't be replicated. Closing ticket.
